src/components/LaboratoryPageComponents/LaboratoryPageContent/hooks/useLaboratoryPageContent.ts

```diff
@@ -4,6 +4,7 @@ import { useNavigate } from 'react-router-dom';
 
 export const useLaboratoryPageContent = (id: string | undefined) => {
   const [isModalVisible, setModalVisible] = useState(false);
+  const [modalType, setModalType] = useState<'edit' | 'delete' | null>(null);
```

Comment:
Для вот таких частных типов лучше использовать type. Только их нужно писать до объявления хука.

```suggestion
  type ModalType = 'edit' | 'delete' | null
  const [modalType, setModalType] = useState<ModalType>(null);
```